man/oci-runtime-tool-generate.1.md

```diff
@@ -102,6 +102,9 @@ read the configuration from `config.json`.
 **--linux-cpus**=CPUS
   Sets the CPUs to use within the cpuset (default is to use any CPU available).
 
+**--linux-hugepape-limit**=[]
+  Limit hugepage resource e.g. --linux-hugepage-limit=2MB:102400
```

Comment:
You should point out that this option can be listed multiple times and that each instance appends a new limit to the generated config.  It would also be nice to have a way to clear the field, but folks can always use `jq` for that (or any of these manipulations ;).